Source/EventFlow.Elasticsearch.Tests/IntegrationTests/ElasticsearchReadModelStoreTests.cs

```diff
@@ -92,37 +92,47 @@ private void PrepareIndexes(IRootResolver resolver)
                 var esType = readModelType.GetTypeInfo()
                     .GetCustomAttribute<ElasticsearchTypeAttribute>();
 
-                var aliasResponse = _elasticClient.GetAlias(x => x.Name(esType.Name));
+                //var indexName = GetIndexName(esType.RelationName);
+                var aliasResponse = _elasticClient.Indices.GetAlias(esType.RelationName);
+                var alias2Response = _elasticClient.GetAliasesPointingToIndex(esType.RelationName);
+                var indexResponse = _elasticClient.Indices.Get(esType.RelationName);
 
                 if (aliasResponse.ApiCall.Success)
                 {
                     if (aliasResponse.Indices != null)
                     {
                         foreach (var indice in aliasResponse?.Indices)
                         {
-                            _elasticClient.DeleteAlias(indice.Key, esType.Name);
-
-                            _elasticClient.DeleteIndex(indice.Key,
-                                d => d.RequestConfiguration(c => c.AllowedStatusCodes((int)HttpStatusCode.NotFound)));
+                            _elasticClient.Indices.DeleteAlias(
+                                indice.Key, 
+                                esType.RelationName
+                            );
+
+                            _elasticClient.Indices.Delete(
+                                indice.Key,
+                                d => d.RequestConfiguration(c => c.AllowedStatusCodes((int)HttpStatusCode.NotFound))
+                            );
                         }
 
-                        _elasticClient.DeleteIndex(esType.Name,
-                            d => d.RequestConfiguration(c => c.AllowedStatusCodes((int)HttpStatusCode.NotFound)));
+                        _elasticClient.Indices.Delete(
+                            esType.RelationName,
+                            d => d.RequestConfiguration(c => c.AllowedStatusCodes((int)HttpStatusCode.NotFound))
+                        );
                     }
                 }
-
-                var indexName = GetIndexName(esType.Name);
+                var indexName = GetIndexName(esType.RelationName);
 
                 _indexes.Add(indexName);
 
-                _elasticClient.CreateIndex(indexName, c => c
+                _elasticClient.Indices.Create(indexName, c => c
                     .Settings(s => s
                         .NumberOfShards(1)
                         .NumberOfReplicas(0))
-                    .Aliases(a => a.Alias(esType.Name))
-                    .Mappings(m => m
-                        .Map(TypeName.Create(readModelType), d => d
-                            .AutoMap())));
+                    .Aliases(a => a.Alias(esType.RelationName)));
+                    // I do not believe this is still needed? If yes, I am unsure how to pass the Type into the mapping method
```

Comment:
If you automatic mapping, it is. I seem to remember that for 7.x you can pass the type. If you can't find it, I'll give it a go

Reply:
@rasmus Happy to update the release notes but I am unsure what exactly you are asking me to change. 

Updated NEST nuget package to 7.8.2 (targets Elastic Search >= 7.8.2) 

Is this what you would like me to change?

Regarding Integration Test image, I do not know what that means or how to change it... could you give me instructions?

Regarding Auto Mapping, I did spend a considerable amount of time trying to pass a generic type but could not find a way to make it work. Your help would be appreciated.

Reply:
@rasmus Quick follow up on my above comments... do you have any guidance so we can finalize this pull request?

Reply:
Sorry, I have been buried in work and haven't had any time to look at EventFlow. I'll have a look at this tonight

Reply:
Could you add a line with something like this to the release notes

- Updated Elasticsearch NEST from v6.1.0 to v7.8.2

As for the un-commented lines in the test around automap, just delete them.

Reply:
Pull request updated with your feedback. Let me know if any other changes required.